docs/pages/2020_Intro_Week/notebooks/bandstructure.md

```diff
@@ -0,0 +1,131 @@
+* Run a non self-consistent field band structure calculation along a path of high symmetry k-points determined by [seekpath](http://materialscloud.org/tools/seekpath)
+
+Numerical parameters for the default 'theos-ht-1.0' protocol are determined as follows:
+* Suitable pseudopotentials and energy cutoffs are automatically searched from the [SSSP library](http://materialscloud.org/sssp) installed on your machine  (it uses the efficiency version 1.1).
+* K-point mesh is selected to have a minimum k-point density of 0.2 Å<sup>-1</sup>
+* A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations
+
+In case the pseudopotentials are not installed, they can be downloaded in a terminal as:
+
+    mkdir sssp_pseudos
+    wget 'https://archive.materialscloud.org/record/file?filename=SSSP_1.1_PBE_efficiency.tar.gz&record_id=23&file_id=d2ce4186-bf76-4e05-8b39-444b4da30273' -O SSSP_1.1_PBE_efficiency.tar.gz
+    tar -C sssp_pseudos -zxvf SSSP_1.1_PBE_efficiency.tar.gz
+    verdi data upf uploadfamily sssp_pseudos 'SSSP' 'SSSP pseudopotential library'
+
+The protocol looks for a UPF file with a specific hash code, that is unique for each different file.
+You can check that you have the right
+one by performing a search in the database:
+
+    UpfData = DataFactory('upf')
+    qb=QueryBuilder()
+    qb.append(UpfData, filters={'attributes.md5':{'==':'cfc449ca30b5f3223ec38ddd88ac046d'}})
+    len(qb.all())
+
+'md5' is a searchable attribute of the pseudopotential data object.
+
+```{code-cell} ipython3
+# This will take approximately 6 minutes on the tutorial AWS (for Al)
+results = run(
+    PwBandStructureWorkChain,
+    code=code,
+    structure=structure_Al
+)
+```
+
+```{code-cell} ipython3
+fermi_energy = results['scf_parameters'].dict.fermi_energy
+results['band_structure'].show_mpl(y_origin=fermi_energy, plot_zero_axis=True)
+
+print("""Final crystal symmetry: {spacegroup_international} (number {spacegroup_number})
+Extended Bravais lattice symbol: {bravais_lattice_extended}
+The system has inversion symmetry: {has_inversion_symmetry}""".format(
+    **results['seekpath_parameters'].get_dict()))
+```
+
+If you want to use a different pseudopotential family (or version of the family) (for instance [SSSP v1.0](https://doi.org/10.24435/materialscloud:2018.0001/v1) instead of the default SSSP v1.1) you can pass an additional parameter when calling the WorkChain, as follows:
+
+    run(
+        # ...,
+        protocol = Dict(dict={
+           'name':'theos-ht-1.0',
+           'modifiers': {
+           'pseudo' : 'SSSP-efficiency-1.0'
+           }
+        })
+    )
+
+(note that only some values are accepted for pseudo, that you can find [here](https://github.com/aiidateam/aiida-quantumespresso/blob/a52266d096afe48dbc6b38b63ac17a9989dd12fe/aiida_quantumespresso/utils/protocols/pw.py#L24); and that you will have to import the pseudopotentials in AiiDA first).
```

Comment:
````suggestion
```{note}
Only some values are accepted for pseudo, that you can find [here](https://github.com/aiidateam/aiida-quantumespresso/blob/a52266d096afe48dbc6b38b63ac17a9989dd12fe/aiida_quantumespresso/utils/protocols/pw.py#L24); and that you will have to import the pseudopotentials in AiiDA first.
```
````